shared/js/initApp.js

```diff
@@ -172,21 +172,64 @@ dashboard.saveProject = function(callback) {
       location.hash = dashboard.currentApp.id = data.id;
       callbackSafe(callback, data);
     });
-    dashboard.showProjectHeader();
+    if (!dashboard.currentApp.hidden) {
```

Comment:
I'm having hidden mean "do not show the project header and do not show the project in the list". for now. We need that only for multilevel embedded projects now, but we'll also use them for freeplay next.